Libplanet/Store/DefaultStore.cs

```diff
@@ -246,174 +271,237 @@ public override IEnumerable<TxId> IterateStagedTransactionIds()
         /// <inheritdoc/>
         public override IEnumerable<TxId> IterateTransactionIds()
         {
-            IEnumerable<string> filenames;
-            _txLock.EnterReadLock();
-            try
+            foreach (UPath path in _txs.EnumerateDirectories(UPath.Root))
             {
-                filenames = _db.FileStorage
-                    .Find(TxIdPrefix)
-                    .Select(file => file.Filename);
-            }
-            finally
-            {
-                _txLock.ExitReadLock();
-            }
+                string upper = path.GetName();
+                if (upper.Length != 2)
+                {
+                    continue;
+                }
 
-            return filenames.Select(filename => new TxId(ByteUtil.ParseHex(filename)));
+                foreach (UPath subPath in _txs.EnumerateFiles(path))
+                {
+                    string lower = subPath.GetName();
+                    if (lower.Length != 62)
+                    {
+                        continue;
+                    }
+
+                    string name = upper + lower;
+                    TxId txid;
+                    try
+                    {
+                        txid = new TxId(ByteUtil.ParseHex(name));
+                    }
+                    catch (Exception)
```

Comment:
Is this `catch` intended? if so, it would be better that we describe comments about why we should catch them.

Reply:
Fixed: d5dadb3e4532dbbe132f9e85832461ad34cbef05.